### PR TITLE
Update main.go to support passing in temp_dir

### PR DESCRIPTION
**Pull request by PhilBrammer**
Allows functionality to pass temp_dir as a mount option in fstab

### CODE REVIEW

**Comment by PhilBrammer** [comment]
Signed CLA

**Comment by jacobsa** [comment]
Thanks for this. Could you please rebase on 6cffeab6847bb43e1c50280f7c941b937b33537f (new `HEAD`) so that Travis will be happy? Sorry for the trouble.

File: tools/mount_gcsfuse/main.go [hunk 1]
```diff
@@ -93,6 +93,14 @@ func makeGcsfuseArgs(
 				"--"+strings.Replace(name, "_", "-", -1),
 				value,
 			)
+			
+		// Special case: support mount-like formatting for gcsfuse string flags.
+		case "temp_dir":
+			args = append(
```
Comment (jacobsa) [inline tools/mount_gcsfuse/main.go]: Instead of duplicating this code, can you fold this into the case below?

**Comment by PhilBrammer** [comment]
No worries on rebasing.  

Consolidated mount options case statement and rebased.

**Comment by jacobsa** [comment]
Thank you very much!

**Pull request merged**